pkg/provider/consulcatalog/consul_catalog.go

```diff
@@ -78,12 +83,15 @@ type EndpointHTTPAuthConfig struct {
 // SetDefaults sets the default values.
 func (p *Provider) SetDefaults() {
 	endpoint := &EndpointConfig{}
-	endpoint.SetDefaults()
 	p.Endpoint = endpoint
 	p.RefreshInterval = ptypes.Duration(15 * time.Second)
 	p.Prefix = "traefik"
 	p.ExposedByDefault = true
 	p.DefaultRule = DefaultTemplateRule
+	p.ConnectAware = false
+	p.ConnectByDefault = false
```

Comment:
```suggestion
```

Reply:
`false` is the value by default for bool in golang, there is no need to explicitly set them to `false`